core/src/components/grid/usage/react.md

```diff
@@ -51,28 +51,28 @@ export const GridExample: React.FC = () => (
       </IonRow>
 
       <IonRow>
-        <IonCol align-self-start>ion-col start</IonCol>
-        <IonCol align-self-center>ion-col center</IonCol>
-        <IonCol align-self-end>ion-col end</IonCol>
+        <IonCol class="ion-align-self-start">ion-col start</IonCol>
```

Comment:
@brandyscarney "class" should be "className" for React.

Reply:
@cdes Ahh good catch! I always forget that.